scripts/security-check.cjs

```
@@ -73,7 +74,8 @@ const SECRET_PATTERNS = [
     id: 'aws-secret-key',
     name: 'AWS Secret Access Key',
     // Require AWS context - look for secret key near aws/AWS keywords or in assignment
-    pattern: /(?:aws|AWS|secret|SECRET|access|ACCESS)[_\-]?(?:secret|SECRET|access|ACCESS|key|KEY)?[_\-]?(?:key|KEY|id|ID)?[\s]*[=:]["']?([A-Za-z0-9/+=]{40})["']?/g,
+    pattern:
+      /(?:aws|AWS|secret|SECRET|access|ACCESS)[_\-]?(?:secret|SECRET|access|ACCESS|key|KEY)?[_\-]?(?:key|KEY|id|ID)?[\s]*[=:]["']?([A-Za-z0-9/+=]{40})["']?/g,
```

ESLint

[error] Unnecessary escape character: \-.

(no-useless-escape)